lib/matplotlib/animation.py

```diff
@@ -1111,10 +1117,15 @@ class to use, such as 'ffmpeg'. If ``None``, defaults to
                 for anim in all_anim:
                     # Clear the initial frame
                     anim._init_draw()
+                frame_number = 0
+                number_frames = sum([a.save_count for a in all_anim])
```

Comment:
I would call this `total_frames`. `frame_number` and `number_frames` are too similar.